src/main/java/com/jwplayer/jwplatform/client/JWplatformClientV2.java

```diff
@@ -0,0 +1,31 @@
+package com.jwplayer.jwplatform.client;
+
+import java.util.Map;
+
+/**
+ * 
+ * @author smurthy Base JWPlatformClient class for V2 providing common
+ *         functionalities shared by V2 clients.
+ */
+public abstract class JWplatformClientV2 {
```

Comment:
Let's rename to `JWPlatformClientV2`. The lower case p looks a little funny. I believe our checkstyle will complain about 3 capital letters in a row but so be it.